bvm/ballerina-runtime/src/test/resources/config_files/ConfigClashingModule3.toml

```diff
@@ -0,0 +1,7 @@
+[rootOrg.test_module.util.foo]
```

Comment:
Lets create issue for chek above cases with CLI values

Reply:
created #29988